TritonSE/LAP-Student-Tracker
# Add students to student tab with backend support (#105)

### Administrative Info

Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Github Issue]-[3-4 word description separated by dashes]`.

### What issue(s) does this branch close?

#105 

### Changes
What changes did you make?

- Added a backend route to `/students` to pull a list of all students (without repeats) and connected students tab front end to route
- Added filtering and ordering functionality to students tab

### Testing

How did you confirm your changes worked?

- `npm run dev`
- Create students objects in the database
- Check that student cards appear for each student object, and they are able to be searched, ordered, and filtered.

### Confirmation of Change

Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| Students Tab with Dummy Students without Levels |
| ----------------------- |
| ![image](https://user-images.githubusercontent.com/23545972/196867330-8754eceb-f15b-43bd-ba51-3687082587e0.png) |
